src/snapred/backend/service/ReductionService.py

```diff
@@ -32,7 +32,7 @@ def reduce(self, runs: List[RunConfig]) -> Dict[Any, Any]:
             # import json
             # data[run.runNumber] = json.dumps(reductionIngredients.__dict__, default=lambda o: o.__dict__)
             try:
-                ReductionRecipe().executeRecipe(reductionIngredients)
+                ReductionRecipe().executeRecipe(ReductionIngredients=reductionIngredients)
```

Comment:
That works fine when a service just does one thing, however it probably wouldnt fly in the CalibrationService.  So it might be reasonable to keep the name parity even if its a little verbose just so it stays in line with the pattern of other files.